src/pages/Home/Search.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { CSSProperties } from "react";
+
+type SearchProps = {
+  onChange(event: React.ChangeEvent<HTMLInputElement>): void;
+}
+
+function Search({ onChange }: SearchProps ): JSX.Element {
+  const backgroundImage = `url(https://images.unsplash.com/photo-1514996937319-344454492b37?ixlib=rb-1.2.1&auto=format&fit=crop&w=3150&q=80)`;
+
+  const blockStyle: CSSProperties = {
+    backgroundImage,
+  }
+
+  return (
+    <div className="flex flex-col text-white items-center justify-center p-4 bg-center bg-cover box-border w-screen" style={blockStyle}>
+    <h2 style={{ fontSize: '2.3rem', marginBottom: '1rem' }}>Busca entre nuestras comunidades</h2>
+      <div className="w-full max-w-xs">
+        <form className="">
+          <div className="mb-4">
+            <input
+              className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+              id="search"
+              type="search"
+              placeholder="Nombre, Lenguaje de Programacion"
```

Comment:
```suggestion
              placeholder="Palabras de busqueda"
```

Reply:
@foursixnine dude regarding this change im not agree with the replacement, as the search expects the user to enter either a programming language or a community name. The term `Palabras de búsqueda` too generic for this use case.

The placeholder should give kind of a hint to the end user, and the term suggested is no so accurate.

Reply:
`Palabras de busqueda` is more accurate, since in the list of channels, it's not just programming languages, but also others areas of technology, sports, etc. Another suggestion could be `Buscar grupos`

Reply:
Hey dude!
I understand what you mean, we have so many things to search, either Group Name, Concepts, maybe topics? But as far as I know we just have the name of the group and the founders/admins of each group.

As we can see this, the main topic is technology, if users add other topics then I think we will havd to improve our data structure to support other topics than technology.

The other thing is the content of the placeholder, placeholders are mean to be hints for the input, not labels. The words that you want to use are labels not hints.

The label of this input is "Busca entre nuestras comunidades", if we want to refer to "What is this input about" then we must change the label and not the placeholder.

Reply:
@EstebanBorai Ok, I'm convinced :)